leaflet-maplibre-gl.js

```diff
@@ -7,7 +7,8 @@
         module.exports = factory(require('leaflet'), require('maplibre-gl'));
     } else {
         // Browser globals (root is window)
-        root.returnExports = factory(window.L, window.maplibregl);
+        root = typeof globalThis !== 'undefined' ? globalThis : root || self;
```

Comment:
overwriting a function argument is principally bad practice. I would prefer to do the check outside the function instead